src/rpc/core_rpc_server.cpp

```diff
@@ -388,6 +388,14 @@ namespace cryptonote
       res.txs_as_hex.push_back(e.as_hex);
       if (req.decode_as_json)
         res.txs_as_json.push_back(e.as_json);
+
+      // output indices too
+      bool r = m_core.get_tx_outputs_gindexs(tx_hash, e.indices);
```

Comment:
I think o_indexes will be a better name instead of indices, it would be like:
`bool r = m_core.get_tx_outputs_gindexs(tx_hash, e.o_indexes);`